src/elements/Header/Subheader.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+import META from 'src/utils/Meta';
+
+export default class Subheader extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+  };
+  static _meta = {
+    library: META.library.semanticUI,
+    name: 'Sub',
+    type: META.type.element,
+  };
+  render() {
+    const classes = classNames(
+      'sd-sub',
+      'sub',
+      this.props.className,
+      'header',
+    );
+    return (
+      <div className={classes}>
+        {this.props.children}
+      </div>
+    );
+  }
+}
```

Comment:
Oh, actually, it needs to spread props.  See `_Header.js` for a reference on how to use `getUnhandledProps()` to spread additional props.